docs/samples/pipelines/kfs-pipeline.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# KFServing Pipeline samples"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Install the necessary kfp library"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip3 install kfp --upgrade"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import kfp.dsl as dsl\n",
+    "import kfp\n",
+    "from kfp import components\n",
+    "import json\n",
+    "\n",
+    "# Create kfp client\n",
+    "# Note: Add the KubeFlow Pipeline endpoint below if the client is not running on the same cluster.\n",
+    "client = kfp.Client('http://169.45.69.232:30905/pipeline')\n",
+    "EXPERIMENT_NAME = 'KFServing Experiments'\n",
+    "experiment = client.create_experiment(name=EXPERIMENT_NAME)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### TensorFlow example\n",
+    "\n",
+    "Note: Change the action from `update` to `create` if you are deploying the model for the first time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kfserving_op = components.load_component_from_url('https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kubeflow/kfserving/component.yaml')\n",
+    "\n",
+    "@dsl.pipeline(\n",
+    "  name='kfserving pipeline',\n",
+    "  description='A pipeline for kfserving.'\n",
+    ")\n",
+    "def kfservingPipeline(\n",
+    "    action = 'update',\n",
+    "    model_name='tf-sample',\n",
+    "    default_model_uri='gs://kfserving-samples/models/tensorflow/flowers',\n",
+    "    canary_model_uri='gs://kfserving-samples/models/tensorflow/flowers-2',\n",
+    "    canary_model_traffic_percentage='10',\n",
+    "    namespace='kubeflow',\n",
```

Comment:
Don't deploy to the kubeflow namespace. This is a system namespace.

Reply:
Done @ellis-bigelow